app/react-components/.eslintrc

```diff
@@ -6,6 +6,7 @@
     "flowtype/require-valid-file-annotation": 0,
     "flowtype/require-return-type": 0,
     "flowtype/require-parameter-type": 0,
+    "flowtype/sort-keys": 0,
```

Comment:
the rule was annoying me because it wants the keys to be ordered alphabetically. Do we care what order keys are in?